### ISSUE TITLE
Add project to PyPI

### ISSUE THREAD

**Issue by dizballanze** [Sagacify/komitet-gita-bezopasnosti#16]
Do you plan to add the project to python package index ? I can help with this task.

**Comment by aborsu** [comment]
We didn't plan on adding it to PyPI as it's not really a library and we use docker in house for deployment, but this is mostly a "no time to do it" problem.
Ideally if we did it, or if you want to do it and merge it into this repository, we would be using circle-ci to deploy it to PyPI automatically once code has been merged into master.
Just as we push the new docker image to the docker hub repository.

**Comment by aborsu** [comment]
Hello, the rules are here: https://github.com/Sagacify/kgb-rules
https://pypi.python.org/pypi/kgb_rules
I will be creating a git commit hook module that will also be added to pypi.
Once that is done, the webhook part (this part) will also be added to pypi.
Should be finished by the end of the week.

**Comment by aborsu** [comment]
The whole project is now in testing here.
https://testpypi.python.org/pypi/kgitb/0.0.6

If we don't find any bugs I will release it on pypi next week.
In the mean time you can install it using 

```
pip install --upgrade -i https://testpypi.python.org/pypi kgitb
```

Checkout #20 for more details.
